src/paths.rs

```diff
@@ -45,6 +45,21 @@ pub struct PathItem {
 }
 
 impl PathItem {
+    pub fn into_iter(self) -> impl Iterator<Item = Operation> {
+        vec![
+            self.get,
+            self.put,
+            self.post,
+            self.delete,
+            self.options,
+            self.head,
+            self.patch,
+            self.trace,
+        ]
+        .into_iter()
+        .flat_map(Option::into_iter)
+    }
+
```

Comment:
I really should have documented this when I added; can you please add a simple doc comment?